configure

```diff
@@ -88,6 +88,11 @@ parser.add_option("--openssl-no-asm",
     dest="openssl_no_asm",
     help="Do not build optimized assembly for OpenSSL")
 
+parser.add_option("--openssl-fips",
```

Comment:
Very minor. Everywhere else in the PR, `'` is used and only here `"` is used. Is there any convention followed?